services/github/src/constants.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  CONCLUSIONS: { // maps status communicated by the queue service to github checkrun conclusions
+    /*eslint quote-props: ["error", "consistent-as-needed"]*/
```

Comment:
ah sorry, I meant that we should add this to our global eslint config. feel free to do it in a followup bug though.